pegged/docs/wiki

```diff
@@ -1 +1 @@
-Subproject commit f111de4928c41cc22d2d093e3362f00bf7f80733
+Subproject commit 69188d442c1e7e4ad3976e5fc3988792723fc097
```

Comment:
What part of the wiki did you change? I had a look in the 'Extended Pegged Syntax' section, but did not find the "abc"i syntax you introduced.